[0m[[0mdebug[0m] [0mRunning Test forcomp.AnagramsSuite : subclass(false, org.scalatest.Suite) with arguments [0m
org.scalatest.exceptions.DuplicateTestNameException: Duplicate test name: sentence anagrams: Linux rulez
	at org.scalatest.SuperEngine.registerTest(Engine.scala:460)
	at org.scalatest.FunSuite$class.test(FunSuite.scala:1212)
	at forcomp.AnagramsSuite.test(AnagramsSuite.scala:11)
	at forcomp.AnagramsSuite.<init>(AnagramsSuite.scala:108)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
	at java.lang.Class.newInstance0(Class.java:357)
	at java.lang.Class.newInstance(Class.java:310)
	at org.scalatest.tools.ScalaTestFramework$ScalaTestRunner.run(ScalaTestFramework.scala:182)
	at org.scalatools.testing.Runner2.run(Runner2.java:16)
	at sbt.TestRunner.delegateRun(TestFramework.scala:57)
	at sbt.TestRunner.run(TestFramework.scala:51)
	at sbt.TestRunner.runTest$1(TestFramework.scala:71)
	at sbt.TestRunner.run(TestFramework.scala:80)
	at sbt.TestFramework$$anonfun$9$$anonfun$apply$9$$anonfun$10$$anonfun$apply$10.apply(TestFramework.scala:188)
	at sbt.TestFramework$$anonfun$9$$anonfun$apply$9$$anonfun$10$$anonfun$apply$10.apply(TestFramework.scala:188)
	at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:200)
	at sbt.TestFramework$$anonfun$9$$anonfun$apply$9$$anonfun$10.apply(TestFramework.scala:188)
	at sbt.TestFramework$$anonfun$9$$anonfun$apply$9$$anonfun$10.apply(TestFramework.scala:188)
	at sbt.Tests$$anonfun$makeParallel$1$$anonfun$apply$7.apply(Tests.scala:119)
	at sbt.Tests$$anonfun$makeParallel$1$$anonfun$apply$7.apply(Tests.scala:119)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:47)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:47)
	at sbt.std.Transform$$anon$5.work(System.scala:71)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:232)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:232)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
	at sbt.Execute.work(Execute.scala:238)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:232)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:232)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[0m[[31merror[0m] [0mCould not run test forcomp.AnagramsSuite: org.scalatest.exceptions.DuplicateTestNameException: Duplicate test name: sentence anagrams: Linux rulez[0m
[0m[[31merror[0m] [0mError: Total 0, Failed 0, Errors 0, Passed 0, Skipped 0[0m
[0m[[31merror[0m] [0mError during tests:[0m
[0m[[31merror[0m] [0m	forcomp.AnagramsSuite[0m
[0m[[31merror[0m] [0m(assignment/test:[31mtest[0m) sbt.TestsFailedException: Tests unsuccessful[0m
